package.json

```diff
@@ -24,19 +24,28 @@
     ]
   },
   "author": {
-    "name": "Chris Sauve",
-    "email": "chris.sauve@shopify.com"
+    "name": "Shopify",
+    "email": "dev@shopify.com"
   },
   "bugs": {
-    "url": "https://github.com/Shopify/javascript/issues"
+    "url": "https://github.com/Shopify/shopify-codemod/issues"
   },
   "license": "MIT",
   "devDependencies": {
-    "chai-jscodeshift": "file:../chai-jscodeshift",
-    "jscodeshift": "^0.3.29"
+    "babel-cli": "6.x",
+    "babel-core": "6.x",
+    "chai": "3.x",
+    "chai-jscodeshift": "15.0.0",
+    "coveralls": "2.x",
+    "eslint-plugin-shopify": "15.1.0",
+    "isparta": "4.x",
+    "istanbul": "0.4.4",
```

Comment:
Fixed on `istanbul@0.4.4` because of this bug: https://github.com/douglasduteil/isparta/issues/126